<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStatusCommentClass</class>
 <widget class="QMainWindow" name="EditStatusCommentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditStatusComment</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Where</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetModeAll">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>241</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetGUIAll">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>481</width>
       <height>192</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>DLLRoot</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DLLName</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>InstName</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Object Path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>831</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>All opinions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetCommentOpinionList">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>51</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameComment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>801</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>コメント・メッセージ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditCommentNow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>781</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frameCommentText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>801</width>
       <height>40</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonLoadHTML">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Load HTML</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRegist">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS UI Gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>登録</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClear">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonFont">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frameBottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>10</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>279</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>8</y>
       <width>671</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>ToolTip</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFontToolTip">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearToolTip">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadHTMLToolTip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Load HTML</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditToolTip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>781</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxEnableToolTip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>6</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
